src/DeviceInterface/AppleOverlayDeviceInterface.js

```diff
@@ -11,12 +13,6 @@ import {createNotification} from '../../packages/device-api/index.js'
  * can be viewed as *not* executing within a regular page context.
  */
 class AppleOverlayDeviceInterface extends AppleDeviceInterface {
-    /**
-     * Mark top frame as not stripping credential data
-     * @type {boolean}
-     */
-    stripCredentials = false;
```

Comment:
There was no need for this, we've never stored passwords anyway, so I've removed this flag.